.travis.yml

```diff
@@ -10,6 +10,7 @@ install:
   - sudo apt-get install libgmp3-dev libdbus-1-dev
   - cpanm -nq --installdeps --with-feature=coverage .
   - gem install sass
+  - (cd /tmp/; git clone https://github.com/inway/mojo-rabbitmq-client.git; cd mojo-rabbitmq-client; perl Makefile.PL; make install)
```

Comment:
the advantage of comments is that you don't have to guess. But you even guessed wrong: http://search.cpan.org/~sebapod/Mojo-RabbitMQ-Client-0.0.6/ (ok, that's mean as he uploaded it today :)